src/components/ProgressBar/index.tsx

```diff
@@ -95,6 +104,66 @@ const ProgressBar: FunctionComponent<ProgressBarProps> = ({
 }) => {
     const classes = useStyles();
 
+    const LinearProgressComponent: React.FunctionComponent<{ value: number; displayValue: boolean; props?: any[] }> = ({
```

Comment:
The external components can move out of the function

Reply:
Done.